animal/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Copyright (C) 2020 Open Source Integrators
```

Comment:
Please remove copyright lines from init files. Also make license lines https: (in all files).